modules/sum-transformers/additional/summary/summary.go

```diff
@@ -0,0 +1,55 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package summary
```

Comment:
At least Package level documentation would be very helpful. 
What is the purpose of this package

Reply:
I have added a task on our board for adding package-level documentation for all our modules